### ISSUE TITLE
Posts: Cannot edit

### ISSUE THREAD

**Issue by davidwessman** [fsek/web#132]
Unable to edit posts, when going into ```/redigera``` no attributes is given or anything, unable to save.

**Comment by davidwessman** [comment]
Couldn't figure this out - any clues why "Title" for example is not showing? 
@fsek/admins

**Comment by jforberg** [comment]
I'll take a look at it.

**Comment by jforberg** [comment]
Ok. So, you need to send an object to ``form_for``, otherwise it doesn't know what object to use for values. Now you have:

```
form_for council_post_path(@council, @post), method: :patch do |f|
```

Note that the argument to form_for is just a plain string, there are no values there. The line should read:

```
form_for @post, :url => council_post_path(@council,@post), :method => :patch do |f|
```

Although even that is too verbose and should be shortened to 

```
form_for [@council, @post] do |f|
```

Which is much clearer, it shows that Post is a nested resource under Council. Method: patch is also unnecessary, patch is the default.

Hope this clears it up a bit.

**Comment by davidwessman** [comment]
Ofcourse! Did not see that, cheers!

**Comment by jforberg** [comment]
Always use the resourceful forms, never plain forms. I always regret it later when I do :)

**Comment by davidwessman** [comment]
The whole Post-module needs a rework... ;)

**Comment by jforberg** [comment]
Yes

**Comment by davidwessman** [comment]
Fixed in #88
